website/versioned_docs/version-11.0.2/complex_parameters.md

```diff
@@ -171,7 +171,7 @@ importKey(Tezos, emailExample, passwordExample, mnemonicExample, secretExample)
     println(`Waiting for ${op.hash} to be confirmed...`);
     return op.confirmation(1).then(() => op.hash);
 }).then(hash => {
-    println(`Operation injected: https://better-call.dev/hangzhou2net/KT1CfFBaLoUrgv93k8668KCCcu2hNDNYPz4L/operations`);
+    println(`Operation injected: https://better-call.dev/ithacanet/KT1CfFBaLoUrgv93k8668KCCcu2hNDNYPz4L/operations`);
```

Comment:
Might be a mistake to change the URL here as the contract address did not change

Reply:
repaired